File: java/src/main/java/io/fivexer/sdk/model/WorkerBreakStarted.java
package io.fivexer.sdk.model;

/**
 * The result of starting a break.
 *
 * <p>Populated by Gson via field reflection and exposed read-only; callers receive instances
 * from the client and never construct them.
 */
public class WorkerBreakStarted {
    private String workerId;
    private boolean onBreak;
    private String since;

    public String getWorkerId() { return workerId; }
    public boolean isOnBreak() { return onBreak; }
    public String getSince() { return since; }
}

File: php/src/Model/WorkerBreakStarted.php
<?php

declare(strict_types=1);

namespace Fivexer\SDK\Model;

/**
 * The result of starting a break.
 *
 * Response model: built from a decoded JSON array and read-only thereafter.
 */
final class WorkerBreakStarted
{
    public function __construct(
        public readonly string $workerId,
        public readonly bool $onBreak,
        public readonly string $since,
    ) {
    }

    /**
     * @param array<string, mixed> $data
     * @internal
     */
    public static function fromArray(array $data): self
    {
        return new self(
            workerId: (string) ($data['workerId'] ?? ''),
            onBreak: (bool) ($data['onBreak'] ?? false),
            since: (string) ($data['since'] ?? ''),
        );
    }
}
